hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -244,6 +250,15 @@ public static HoodieWriteConfig getHoodieClientConfig(
     return writeConfig;
   }
 
+  private static Properties callbackConfigProperties(Configuration conf) {
+    Properties properties = new Properties();
+    Map<String, String> callbackConfig = conf.toMap().entrySet().stream()
```

Comment:
Can we just return the callback config instead ?

Reply:
Can we reuse `FlinkOptions.getPropertiesWithPrefix` ?

Reply:
@danny0405 resolved with reuse FlinkOptions.getPropertiesWithPrefix ?